dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubHelper.java

```diff
@@ -42,10 +47,50 @@
 public final class GitHubHelper {
 
     private static final Logger LOG = LoggerFactory.getLogger(GitHubHelper.class);
+    public static final String BRANCHNAME_FOR_BOT = "feature/add_dockstore_yml";
+
+    public static final String DOCKSTORE_BOT_PR_TEXT = """
+           Dockstore's bot has guessed at a .dockstore.yml for your repository.
+           
+           Please review, make appropriate changes, add any [additional keys](https://docs.dockstore.org/en/stable/assets/templates/template.html) and merge in order to complete integration with Dockstore, allowing Dockstore to keep informed of new changes to your workkflow, notebook, or tools.
+           """;
 
     private GitHubHelper() {
     }
 
+    /**
+     * Experimental building block, given a repository, create a fork of it and a PR to it
+     * @param inferredDockstoreYml should be the only real change in the PR
+     * @param gitHub    The GitHub API
```

Comment:
Worth clarifying who this is; I was confused because I was trying to figure out under what namespace the fork gets created. This would typically  be the dockstore-testing user (bot?), right?

Is the ultimate idea that we would create a token for the dockstore-testing user and put it in AWS Secrets Manager?

Reply:
Yes, this would typically be the user that provided the access key. Will add comment. 
Ultimate idea is correct